shared/js/background/dnr-session-rule-id.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * For managing dynamically created MV3 session rules
+ * getNextSessionRuleId will return the next unique session rule id to use when creating new session rules
+ **/
+const browserWrapper = require('./wrapper.es6')
```

Comment:
Nit: Please use ES Module syntax for this import.
```suggestion
import * as browserWrapper from './wrapper.es6'
```